fog/overseer/server/src/bin/main.rs

```diff
@@ -0,0 +1,42 @@
+// Copyright (c) 2018-2022 The MobileCoin Foundation
+
+use mc_common::logger::{log, o};
+use mc_fog_overseer_server::{config::OverseerConfig, server, service::OverseerService};
+use mc_fog_sql_recovery_db::SqlRecoveryDb;
+use structopt::StructOpt;
+
+/// Starts a Rocket server that allows clients to access Fog Overseer APIs
+/// over HTTP.
+fn main() {
+    mc_common::setup_panic_handler();
+    // TODO: Enable sentry here.
```

Comment:
Why not just stick that piece of code here? That's all that is needed to enable it.

Reply:
I haven't set up sentry alerting yet and would like to do it all in one PR.